templates/_includes/article-items.html

```diff
@@ -5,13 +5,13 @@
 {% else %}
   {% for item in items %}
   <li class="p-list__item">
-    <a href="{{ item.link }}">{{ item.title }}</a><br />
+    <a href="{{ item.link | replace_admin }}">{{ item.title.rendered | replace_encoded_endash }}</a><br />
```

Comment:
You can replace `replace_encoded_endash` with `safe` to stop Django from auto-escaping encoded characters
https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#std:templatefilter-safe